index.js

```diff
@@ -1,38 +1,71 @@
-function * entries (obj) {
-  for (let key of Object.keys(obj)) {
-    yield [obj[key], key]
-  }
-}
+'use strict'
```

Comment:
Can't find any documentation/release notes stating so, but I still feel for the sake of setting a good example its better to have.